### ISSUE TITLE
snapshot always updated even without --snapshot-update

### ISSUE THREAD

**Issue by emilyzzz** [syrusakbary/snapshottest#24]
I'm running py.test with snapshottest, for a django+graphql project. The problem is, even when I do not pass in "py.test --snapshot-update", the snapshot dir still gets updated with new result, and py.test will always output 'successful'.

Versions
```
$ python --version
Python 3.6.3

$ pip show pytest
Name: pytest
Version: 3.2.0

$ pip show snapshottest
Name: snapshottest
Version: 0.5.0
```

To reproduce:
```
# first delete all saved snapshots
rm -fr mypath/tests/snapshots/

# simple test graphql user query
def test_users_query(snapshot):
    query = '''{ users { id email }}'''
    result = client.execute(query)
    #result = None
    snapshot.assert_match(result)

# first time run py.test, expect 'snapshots' saved, got successful pytest run.
$ py.test

# check saved snapshot, it has 920 characters
# cat mypath/tests/snapshots/snap_test_query.py | wc -c
920

# modify test to uncomment "#result = None" as seen above, expect 'assert' to fail
$ py.test

# output from py.test, please ignore the 2nd test since that's irrelavent
$ py.test
Test session starts (platform: darwin, Python 3.6.3, pytest 3.2.0, pytest-sugar 0.8.0)
benchmark: 3.1.1 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /Users/me/me/mypath, inifile:
plugins: xdist-1.18.2, sugar-0.8.0, cov-2.5.1, benchmark-3.1.1, nameko-2.7.0, snapshottest-0.5.0

 mypath/tests/test_query.py ✓✓                                               100% ██████████
==================================== SnapshotTest summary =====================================
2 snapshots passed.
2 snapshots written in 1 test suites.

Results (0.92s):
       2 passed

# now found out snapshot updated, with without "--snapshot-update" argument passed in
# after 2nd run, saved snapshot reduced from 920 char to 573 char, basically a dict data structure got replaced with "None" as from result=None. 
$ cat mypath/tests/snapshots/snap_test_query.py | wc -c
     573
```

**Comment by emilyzzz** [comment]
I found the bug and the fix.  Not sure the code contribution procedure on this repo, should I just make a PR? 

it's this line: 
https://github.com/syrusakbary/snapshottest/blob/master/snapshottest/module.py#L27

Currently 
```
        self.imports['snapshottest'].add('Snapshot')
```

Change to
```
        self.imports['snapshottest.snapshot'].add('Snapshot')
```

This will fix the bug.

**Comment by lawrlee** [comment]
I've encountered this issue too and it is a quite a pain. Seems like a simple merge. Can we get the fix in?

**Comment by mprostock** [comment]
I have had the same issue, all snapshots always got overwritten and passed the test even though they clearly should not have. 

For my problem the issue was easy to fix, see my PR from today (https://github.com/syrusakbary/snapshottest/pull/35).

You could be having different issues though, but that should be easy to find:
The problem is line 34 of module.py, where the snapshots are being loaded. The try/except here catches all errors and responds to that by simply acting as if no snapshots are present. So what is meant to be caught here is only the exception of the missing files, but there may be other issues (as in my case issues with file encoding on windows). Insert a "raise" here and find out what your problem actually is.

This is quite problematic because due to this, when any error occurs here, the tests always show as "passed", I think the default behaviour should rather be to raise the exception (thereby causing pytest to show it als "failed" and "error" it it is anything other than the missing file (IOError Exception).

My PR is just a quick fix for the encoding issue, I did not change the "problematic" part of the code itself, might do if PR gets accepted.

**Comment by davidshepherd7** [comment]
Looks like this was fixed by #45
